extensions/azure-keyvault/deployment/src/main/java/io/quarkiverse/azure/keyvault/secret/deployment/KeyVaultSecretProcessor.java

```diff
@@ -10,13 +10,11 @@
 import com.azure.json.JsonSerializable;
 
 import io.quarkiverse.azure.keyvault.secret.runtime.KeyVaultSecretClientProducer;
+import io.quarkiverse.azure.keyvault.secret.runtime.config.KeyVaultSecretConfigBuilder;
 import io.quarkus.arc.deployment.AdditionalBeanBuildItem;
 import io.quarkus.deployment.annotations.BuildProducer;
 import io.quarkus.deployment.annotations.BuildStep;
-import io.quarkus.deployment.builditem.CombinedIndexBuildItem;
-import io.quarkus.deployment.builditem.ExtensionSslNativeSupportBuildItem;
-import io.quarkus.deployment.builditem.FeatureBuildItem;
-import io.quarkus.deployment.builditem.IndexDependencyBuildItem;
+import io.quarkus.deployment.builditem.*;
```

Comment:
I think we'd better not use builditem.*;

Reply:
Thanks for your feedback @backwind1233 . However, we already use `builditem` in other parts of this system. If we are to move away from the use of `builditem` it must be done in a comprehensive way. Can you please create a User Story to investigate this and make a recommendation? 

In the meantime, please proceed with this @galiacheng .

Reply:
See https://github.com/quarkiverse/quarkus-azure-services/pull/266/commits/6037b35e972d07531e6e6d21ee28c125012eae06

Reply:
@backwind1233 We use lots of sub items of `io.quarkus.deployment.builditem`, it's reasonable to use this style here.